sonar-kotlin-plugin/src/main/java/org/sonarsource/kotlin/checks/BiometricAuthWithoutCryptoCheck.kt

```diff
@@ -36,7 +36,7 @@ private val AUTHENTICATE_FUN_MATCHERS = listOf(
     FunMatcher(qualifier = "androidx.biometric.BiometricPrompt", name = "authenticate")
 )
 
-private const val MESSAGE = "Make sure performing a biometric authentication without a CryptoObject is safe here"
+private const val MESSAGE = "Make sure performing a biometric authentication without a CryptoObject is safe here."
```

Comment:
Do we need `""` around the `CryptoObject` here?
```suggestion
private const val MESSAGE = "Make sure performing a biometric authentication without a "CryptoObject" is safe here."
```